terraform/modules/database/variables.tf

```diff
@@ -22,7 +22,7 @@ variable "db_username" {
 
 variable "db_password" {
   type    = "string"
-  default = "examplesofcoolanimals"
+  default = "example"
```

Comment:
I would remove the default here and just require that it's filled out manually. This is more secure.

Reply:
👍

Reply:
👍